src/main/java/com/mycompany/myapp/service/reservation/ReservationService.java

```diff
@@ -48,6 +52,10 @@ public void deleteReservation(Long id) {
         return pool.getReservations();
     }
 
+    public void sendToolsRequest(String poolId, String[] selectedTools) throws Exception{
```

Comment:
By passing an Array here rather than `List<String>` in `sendToolsRequestEmail` you have to make a conversion to list. Just pass a list.